server/methods/addRoomOwner.coffee

```diff
@@ -1,11 +1,12 @@
 Meteor.methods
 	addRoomOwner: (rid, userId) ->
-		unless Meteor.userId()
-			throw new Meteor.Error 'error-invalid-user', 'Invalid user', { method: 'addRoomOwner' }
 
 		check rid, String
 		check userId, String
 
+		unless Meteor.userId()
+			throw new Meteor.Error 'error-invalid-user', 'Invalid user', { method: 'addRoomOwner' }
```

Comment:
@graywolf336  I think its cheaper for the system to check first if the parameters are in their  correct type than to check if the user is valid, but i am not sure.